src/pipeline/scraper.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import os
+import time
+
+from dotenv import load_dotenv
+from firecrawl import Firecrawl
+
+load_dotenv()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def clean_headnote(text: str) -> str:
+    """
+    Removes headnote from scraped content from website
+
+    :param text: text to clean
+    :type text: str
+    :return: cleaned text
+    :rtype: str
+    """
```

Comment:
Trzymajmy się googlowskiego stylu docstringów, czyli:
```suggestion
def clean_headnote(text: str) -> str:
    """
    Removes headnote from scraped content from website. 
    Args:
        text (str): Text to clean. 
    Returns:
        str:  Cleaned text. 
    """
```